### PR TITLE
feat: variables and glossary

### PR DESCRIPTION
**Pull request by kellyjosephprice**
| [![PR App][icn]][demo] | Fix RM-9018 RM-9022 |
| :--------------------: | :-----------------: |

## 🧰 Changes

Supports Variables and GlossaryItems!

Ok, so this PR only has a small 🤏 code change for those components. We just have to format the correct jsx in the editor for this to work, _I think_.

This PR does

- rename tests from `flavored-*` to `*`
- removes `yaml` from the top of the doc
- runs the variable browser test

## 🧬 QA & Testing

- [Broken on production][prod].
- [Working in this PR app][demo].

[demo]: https://markdown-pr-PR_NUMBER.herokuapp.com
[prod]: https://SUBDOMAIN.readme.io
[icn]: https://user-images.githubusercontent.com/886627/160426047-1bee9488-305a-4145-bb2b-09d8b757d38a.svg

### CODE REVIEW

File: example/Form.tsx [hunk 1]
```diff
@@ -21,11 +21,13 @@ const Form = () => {
 
   const onSelect = (event: React.ChangeEvent<HTMLSelectElement>) => {
     setSearchParams(params => {
-      params.delete('edit');
+      console.log(params);
       return params;
     });
 
-    navigate(event.target.value);
+    console.log(event.target.value);
```
Comment (trishaprile) [inline example/Form.tsx:28]: 👀

File: docs/variable-tests.md [hunk 2]
```diff
@@ -1,16 +1,17 @@
...
 ```
-<<defvar>>
+<Variable variable="defvar" />
 ```
 
 and
 
 ```js
-const xyz = "<<defvar>>";
+const xyz = "<Variable variable="defvar" />";
```
Comment (trishaprile) [inline docs/variable-tests.md:16]: should we also add an example for `<GlossaryItem />` to this?

**Review by trishaprile — COMMENTED** [review]
niceee
one behavior I noticed is that `<<` breaks the page

**Comment by kellyjosephprice** [comment]
> niceee one behavior I noticed is that `<<` breaks the page

Yea, not sure what to do with parsing errors yet.

I **do not** want to support the old syntax since it causes a parsing error, that just sounds like a bad time.

**Comment by kellyjosephprice** [comment]
I'm going to merge this, but if you have any other reviews please let me know!

**Comment by rafegoldberg** [comment]
## This PR was released!

🚀 Changes included in [<b>v6.75.0-beta.27</b>](/readmeio/markdown/releases/tag/v6.75.0-beta.27)
<br>

**Pull request merged**